cpp/src/IceGrid/AdminI.cpp

```diff
@@ -19,61 +19,88 @@
 #include <IceGrid/SynchronizationException.h>
 
 using namespace std;
+using namespace std::chrono;
 using namespace Ice;
 using namespace IceGrid;
 
-namespace IceGrid
+namespace
 {
 
 class ServerProxyWrapper
 {
 public:
 
-    ServerProxyWrapper(const DatabasePtr& database, const string& id) : _id(id)
+    ServerProxyWrapper(const shared_ptr<Database>& database, const string& id) : _id(id)
     {
         try
         {
-            _proxy = database->getServer(_id)->getProxy(_activationTimeout, _deactivationTimeout, _node, false, 5);
+            _proxy = database->getServer(_id)->getProxy(_activationTimeout, _deactivationTimeout, _node, false, 5s);
```

Comment:
Seems that `_node` member is never set

Reply:
It's set by `getProxy()`

Reply:
The `_activationTimeout` and `_deactivationTimeout` aren't set either

Reply:
Also set by `getProxy()`